src/MollieApiClient.php

```
@@ -62,7 +63,7 @@ class MollieApiClient
     const HTTP_PATCH = "PATCH";
 
     /**
-     * @var \Mollie\Api\HttpAdapter\MollieHttpAdapterInterface
+     * @var Guzzle6And7MollieHttpAdapter
```

Just dived into the code, I think this still should be a `MollieHttpAdapterInterface`

---

PHP Stan 
`  455    Call to an undefined method Mollie\Api\HttpAdapter\MollieHttpAdapterInterface::enableDebugging().   
  475    Call to an undefined method Mollie\Api\HttpAdapter\MollieHttpAdapterInterface::disableDebugging().  
`

---

After I replace with `Guzzle6And7MollieHttpAdapter` with `MollieHttpAdapterInterface`

---

I'm ok with suppressing these errors in the php stan neon file.

`455 Call to an undefined method Mollie\Api\HttpAdapter\MollieHttpAdapterInterface::enableDebugging(). 475 Call to an undefined method Mollie\Api\HttpAdapter\MollieHttpAdapterInterface::disableDebugging().`